File: kyw0716/level0/OX퀴즈.js
const calculator = (num1, num2, operator) => {
  if (operator === "-") return Number(num1) - Number(num2);
  return Number(num1) + Number(num2);
};

function solution(quiz) {
  var answer = [];
  const elements = [];

  for (let i = 0; i < quiz.length; i++) {
    elements.push(quiz[i].split(" "));
  }

  elements.forEach((element) => {
    if (calculator(element[0], element[2], element[1]) === Number(element[4]))
      answer.push("O");
    else answer.push("X");
  });

  return answer;
}

// https://school.programmers.co.kr/learn/courses/30/lessons/120907

File: sk000801/입문/OX퀴즈.java
class Solution {
    public String[] solution(String[] quiz) {
        String answer = "";
        for(String s : quiz) {
            String[] sik = s.split(" ");
            int result = Integer.parseInt(sik[0]);
            for(int i = 0; i < sik.length; i++) {
                if(sik[i].equals("-")) result -= Integer.parseInt(sik[i+1]);
                if(sik[i].equals("+")) result += Integer.parseInt(sik[i+1]);
            }
            if (String.valueOf(result).equals(sik[sik.length-1])) answer += "O";
            else answer += "X";
        }
        return answer.split("");
    }
}